tests/console/yast2_nfs_client.pm

```diff
@@ -59,14 +59,23 @@ sub run() {
     #
     # Check the result
     #
-    script_run "cat /tmp/nfs/client/file.txt > /dev/$serialdev", 0;
-
+    # check if nfs is mounted
+    script_run 'mount|grep nfs';
     # Wait for more than 90 seconds due to NFSD's 90 second grace period.
-    wait_serial('It worked', 100) || die "Reading from nfs failed.";
+    diag 'waiting 90 second due NFS grace period';
+    sleep 90;
```

Comment:
there must be a better way, how about checking in a loop with early exit on success? Where do the 90 second grace period in nfsd exactly come from? is it a bug?